cmd/titus-log-uploader/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1-alpine as builder
```

Comment:
Why is this a separate builder / Dockerfile than the other ones?

Reply:
Temporary build solution until I got an answer to the standard practice of releasing sidecars.  I'm thinking it might make sense to have this be a separate repo and just pull in the packages I need from this, but was waiting until I got feedback from Rob.

Reply:
If the log uploader is going to live in this repo, then let's move it to `hack/images/titus-log-uploader` (or similar) - that way it matches what we do with sshd.